tests/integration/targets/project/tasks/create.yml

```diff
@@ -0,0 +1,81 @@
+---
+# Copyright: Contributors to the Ansible project
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+- name: Create project in check mode
+  ansible.eda.project:
+    controller_host: "{{ controller_host }}"
+    controller_username: "{{ controller_user }}"
+    validate_certs: "{{ validate_certs }}"
+    name: Example
```

Comment:
I suppose it will be good to use unique names for the resources for the tests, to avoid problems if they are executed in the CI in parallel.